test/integration/buildscript_int_test.go

```diff
@@ -39,6 +48,124 @@ func (suite *BuildScriptIntegrationTestSuite) TestBuildScript_NeedsReset() {
 	suite.Require().FileExists(filepath.Join(ts.Dirs.Work, constants.BuildScriptFileName), ts.DebugMessage(""))
 }
 
+func (suite *BuildScriptIntegrationTestSuite) TestBuildScript_IngredientFunc() {
+	suite.T().Skip("Please enable once ingredient publishing via buildscript on the API is live")
```

Comment:
Could we link this to a story? Even if it's a while before that story gets resolved.